### ISSUE TITLE
Tracking SRXL2 support in PX4

### ISSUE THREAD

**Issue by podhrmic** [camdeno/F16Capstone#11]
PX4 issue: https://github.com/PX4/PX4-Autopilot/pull/17555

A receiver such as: https://www.horizonhobby.com/product/ar637t-dsmx-6-channel-as3x-and-safe-telemetry-receiver/SPMAR637T.html#

Datasheet: https://www.horizonhobby.com/on/demandware.static/-/Sites-horizon-master/default/dwb9928800/Manuals/SPMAR637T-Manual-EN.pdf

Possibly an extension cable? https://www.horizonhobby.com/product/x-bus-extension-12-inch/SPMA9580.html

PX4 pinout: http://www.holybro.com/manual/Pixhawk4Mini_Pinouts.pdf

**Comment by teyrana** [comment]
Yes, the 637T is expected to be supported by this patch.   We will post when we have code suitable for testing.  Stay tuned :)
